### PR TITLE
Added descriptions in JSON example response

### PR DESCRIPTION
**Pull request by dvh**
TODO:

- Remove last \n after markdown transformation (now every \n is being removed)
- Escape { and } in description as { and } are delimiters for internal use

### CODE REVIEW

File: src/helpers/buildExample.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var _ = require('lodash');
+var marked = require('marked');
```
Comment (joostfarla) [inline src/helpers/buildExample.js]: samenvoegen in 1 statement:

```
var _ = require('lodash'),
  marked = require('marked');
```

File: src/helpers/buildExample.js [hunk 2]
```diff
@@ -8,8 +9,18 @@ module.exports = function (schema) {
 
 function traverseSchema (schema) {
   if (schema.type === 'object') {
-    return _.mapValues(schema.properties, function (property) {
-      return traverseSchema(property);
+    return _.transform(schema.properties, function (result, property, key) {
+
```
Comment (joostfarla) [inline src/helpers/buildExample.js]: Witregel weg.

File: src/helpers/buildExample.js [hunk 3]
```diff
@@ -8,8 +9,18 @@ module.exports = function (schema) {
...
+      if (property.description) {
+        key = key + '{' + marked(property.description.replace(/({|})/g, '\\$1')).replace('\n', '') + '}';
+
```
Comment (joostfarla) [inline src/helpers/buildExample.js]: Witregel 1 regel naar onder?

File: templates/_layout.jade [hunk 4]
```diff
@@ -17,3 +17,5 @@ html
           p This documentation is generated by <a href="https://github.com/apiwise/apiranto">Apiranto</a>.
     script(src='https://code.jquery.com/jquery-2.1.4.min.js')
     script(src='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js')
+    script.
```
Comment (joostfarla) [inline templates/_layout.jade]: Punt kan weg (dat is voor classes toevoegen)

Comment (dvh) [inline templates/_layout.jade]: Neuj

**Pull request merged**